### PR TITLE
Add `Schedule.to_generator`

### PR DESCRIPTION
**Pull request by danieldk**
## Description

This method turns a `Schedule` into a generator by feeding the `Schedule` steps with a given starting step and increment.

### Types of change

Feature

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Review by shadeMe — APPROVED** [review]

File: thinc/schedules.py [hunk 1]
```diff
@@ -41,6 +42,27 @@ def attrs(self):
...
         return self._attrs
 
+    def to_generator(
+        self, start: int = 0, step_size=1, **extra
+    ) -> Generator[OutT, None, None]:
+        """Turn the schedule into a generator.
+
+        start (int): The schedule initial step.
+        step_size (int): The amount to increase the step for each generated value.
+        **kwargs: Additional arguments that are passed to the schedule.
```
Comment (shadeMe) [inline thinc/schedules.py:52]:
```suggestion
        **extra: Additional arguments that are passed to the schedule.
```

File: website/docs/api-schedules.md [hunk 2]
```diff
@@ -94,6 +94,27 @@ assert schedule(1000) == 0.1
...
+
+g = constant(0.1).to_generator()
+assert next(g) == 0.1
+assert next(g) == 0.1
+```
+
+| Argument    | Type                                 | Description                                                    |
+| ----------- | ------------------------------------ | -------------------------------------------------------------- |
+| `start`     | <tt>int</tt>                         | The initial schedule step.                                     |
+| `step_size` | <tt>int</tt>                         | The amount to increase the step with for each generated value. |
```
Comment (svlandeg) [inline website/docs/api-schedules.md:114]: These could mention the default values.

Comment (danieldk) [reply website/docs/api-schedules.md]: Added.

File: website/docs/api-schedules.md [hunk 3]
```diff
@@ -94,6 +94,27 @@ assert schedule(1000) == 0.1
 | `**kwargs`  |              | Optional arguments passed to the schedule. |
 | **RETURNS** | <tt>Any</tt> | The schedule value for the step.           |
 
+### Schedule.to_generator {#call tag="method"}
```
Comment (svlandeg) [inline website/docs/api-schedules.md:97]:
```suggestion
### Schedule.to_generator {#to_generator tag="method"}
```

**Review by svlandeg — APPROVED** [review]
Good to merge!

**Pull request merged**